build/jobs/run-tests.yml

```diff
@@ -3,16 +3,9 @@ parameters:
   type: string
 - name: keyVaultName
   type: string
-- name: runIntegration
-  type: boolean
-  default: true
-- name: runE2E
-  type: boolean
-  default: true
```

Comment:
For my learning, why were these parameters taken out?

Reply:
They weren't being used, so I removed them for readability. When this script was first written they were added to help with testing, but there is now no use for running only one type of test.

Reply:
Ah I see - so the conditions on line 15 and 55 would always be true?